sys/net/gnrc/network_layer/ipv6/nib/nib_abr.c

```diff
@@ -89,13 +89,14 @@ bool gnrc_ipv6_nib_abr_iter(void **state, gnrc_ipv6_nib_abr_t *entry)
 void gnrc_ipv6_nib_abr_print(gnrc_ipv6_nib_abr_t *abr)
 {
     char addr_str[IPV6_ADDR_MAX_STR_LEN];
-    uint32_t now = evtimer_now_min();
+    /* Something seems strange here does it expire if valid_until is 0 or not */
```

Comment:
This line needs to be removed i think there is something strange in the code (not of this print but oft he whole thing this print is just the tip of it, the strangeness is not modified by this PR in anyway it is just this line making a notice)

(there are mechanisms that take no input as 10000 sec but do not set the valid date in 10000 secs but set it to  0, than this prints 10000, even though some time might have passed)

Reply:
@benpicco  are you ok with leaving that comment line in?